### PR TITLE
Feature/pc console overview

### LINKED ISSUE #408: PC Console: error refreshing console when the "Reviewer Status" tab is enabled
When I go to the PC Console: 

http://localhost:3000/group?id=AKBC.ws/2019/Conference/Program_Chairs#reviewer-status

refreshing the browser, the count of reviews entered is zero. 

Screen Shot 2019-09-06 at 11 53 31 AM

but when I go to "Paper Status" and then go  back to the tab again I see the correct data:

Screen Shot 2019-09-06 at 11 55 08 AM

### LINKED ISSUE #563: PC Console: show review/meta-review progress
Request from ECCV PCs:

Show al the top of the page the progress of the reviews and meta-reviews. This can be easily done because we have all the data loaded.

### LINKED ISSUE #598: PC Console: show total number of active submission
There is no way for the PCs to know how many active submission are.

### PR DESCRIPTION
**Pull request by zbialecki**
Updates Configuration tab to display a variety of useful statistics on the overall progress of the conference.

Note: when updating an existing PC console webfield using the group editor, make sure to add `WITHDRAWN_SUBMISSION_ID` and `DESK_REJECTED_SUBMISSION_ID` to the constants section.

Fixes #598
Fixes #563 
Fixes https://github.com/openreview/openreview/issues/1213
Fixes #408

### CODE REVIEW

**Comment by melisabok** [comment]
Currently ECCV status:

Screen Shot 2020-03-23 at 5 37 07 PM


I really like this change, it gives more information about the status of the conference. 

All the numbers are associated to an invitation. Another approach is too a show the status of each invitation independently building like a workflow. Like:

* Recruitment: invited reviewers, reviewers, ac,
* Submission: active submissions, desk reject, withdraw
* Bid: status of completion
* Recommendation: status of completion.

All the invitations with their start, due and exp date. Render differently the ones that are already expired, active or be active in the future. 

About the Review stage, I would show the number of reviews submitted, sometimes it takes time to complete all the reviews for a specific reviewers. 

# All Reviews
# Papers with reviews completed.
# Reviewers with reviews completed.

**Comment by zbialecki** [comment]
OK, I can improve the stats on reviews to include the 3 bullets that you suggested. 

For recruitment stats, are you suggesting adding a new row of stats at the top for # reviewers invited/accepted, and # ACs invited/accepted? Or are you suggesting a different layout for all of these stats? How would I get the number of reviewers invited?

**Comment by melisabok** [comment]
I'm suggesting to change the layout where we can group the data per invitation/stage. But this is a suggestion! I'm not sure we should experiment with different layouts. 

With this version we are showing the numbers/stats at the top and the timeline at the bottom, where they are highly related. 

Invited/Declined are groups where you can check the members length. You can see the links in Venue Roles.

**Comment by zbialecki** [comment]
I added the stats you suggested and changed the layout slightly to highlight the different stages more. Here's what it looks like with some out-of-date ICLR data:

Screen Shot 2020-03-26 at 11 35 30 PM

**Comment by mohituniyal** [comment]
I love these changes @zbialecki :)

**Comment by melisabok** [comment]
It looks better!

I still think we should try to show stats and timeline all together group by invitation/stage. And highlight only the stage that is active otherwise you can get lost with too many numbers. It should be one line per invitation, I guess we should use a sketch tool to try this. 

There is a typo in Paper progress: '% of papers that have reveived reviews from all assigned reviewers'. 

In 'Review Submitted' we could know how many reviews we expect to have. I guess counting the number of AnonReviewer groups.

**Comment by zbialecki** [comment]
How do you get the current stage of a venue? Also, what do you mean by 1 line per invitation?

**Comment by melisabok** [comment]
Checking the cdate, duedate and expdate of each invitation. If now is between cdate and expdate then the stage is active. 

I think something like the timeline, where we have one invitation per line plus adding the stats next to each line. Again, I'm not very sure about this, I like your version. We could ask the PCs for more feedback.

**Comment by zbialecki** [comment]
OK, I think I'm starting to understand what you're describing. I do think that would be cool, but I think there would have to be multiple invitations per stage/row of stats. For example, the recruitment stage has Recruit_Reviewers and Recruit_Area_Chairs. I think getting some feedback from actual PCs at this stage would be very helpful.

**Comment by melisabok** [comment]
Some feedback:

- I guess this is happening because is diving by zero:

Screen Shot 2020-03-31 at 9 43 19 AM

- Can we show the invitation description in a single line, may be reduce the timezone string?

Screen Shot 2020-03-31 at 9 43 31 AM

- I'm getting an error when I try to assign a reviewer to a paper:

Screen Shot 2020-03-31 at 9 45 19 AM

- I'm also noticing that `SHOW_AC_TAB` is not being used anymore. This variable specifies if the venue has area chairs or not. I think we should keep using it to avoid extra network calls and hide the AC status tab.

**Comment by zbialecki** [comment]
Thanks for testing Melisa. Those issues are now fixed. About `SHOW_AC_TAB`: that constant is now used to turn off the AC tab and some data specific to that tab, but to truly stop all AC data from being loaded it's necessary to set the `AREA_CHAIRS_ID` constant to be empty.

**Comment by melisabok** [comment]
So, let's remove `SHOW_AC_TAB` constant and use `AREA_CHAIRS_ID` to show/hide the tab.

**Comment by melisabok** [comment]
This is great!!!

Now that I remember Reviewers and ACs have reminder functionality, why do we have reminders in the Paper Status tab?

Screen Shot 2020-04-01 at 1 26 29 PM

This is the option that sends one message request per reviewer/paper. Should we remove it?

**Comment by zbialecki** [comment]
That option actually sends one request per paper (could be multiple reviewers), so it does group the requests when it can. I would prefer to keep it because that has some nice features like allowing you to select individual papers to message, and also includes the review link in the email.

**Comment by melisabok** [comment]
ok, let's keep it. It can be confusing for the PC too many options to remind reviewers. 

I found a bug in the reminder, we should remind reviewers/acs that have assigned papers and missing reviews/meta-reviews:

Screen Shot 2020-04-01 at 1 43 55 PM

Looking at the screenshot we should remind to 'Reviewer ECCV_One' and 'Reviewer ECCV_Two'.

**Comment by zbialecki** [comment]
That should be fixed now, let me know if you see any other issues.

File: openreview/conference/templates/programchairWebfield.js [hunk 1]
```diff
@@ -1453,10 +1453,10 @@ var displayAreaChairsStatusTable = function() {
         return row.summary.completedRecs === 0;
       },
       'msg-unsubmitted-reviews': function(row) {
-        return row.reviewProgressData.numCompletedReviews === 0;
+        return row.reviewProgressData.numCompletedReviews === 0 && row.reviewProgressData.numPapers > 0;
```
Comment (melisabok) [inline openreview/conference/templates/programchairWebfield.js:1456]: I think this should be:

```
return row.reviewProgressData.numCompletedReviews < row.reviewProgressData.numPapers
```

A reviewer can have 2 papers assigned and entered one review and we should remind them.

Comment (zbialecki) [reply openreview/conference/templates/programchairWebfield.js:1456]: Ah you're totally right

**Pull request merged**